Functions/Administration/xASL_adm_GetPhilipsScaling.m

```diff
@@ -29,45 +30,59 @@
 rescaleSlopeNifti = header.dat.scl_slope;
 
 if isfield(parms,'RWVSlope')
-	% If the RealWorldValue is present, then dcm2nii scales to them and ignores everything else
-    fprintf('=========================================== Scaling ==========================================\n');
-	if length(parms.RWVSlope)>1
-		[~,idx] = min(abs(parms.RWVSlope - rescaleSlopeNifti));
-		parms.RWVSlope = parms.RWVSlope(idx);
-		if isfield(parms,'RescaleSlopeOriginal') && length(parms.RescaleSlopeOriginal)>1
-			parms.RescaleSlopeOriginal = parms.RescaleSlopeOriginal(idx);
+	% Obtains dcm2niix version used for conversion and ExploreASL version
+	% If ExploreASL version >= 1.10.0 or dcm2niix version >= 20210317
+	if isfield(parms,'ConversionSoftware') && (strcmp(parms.ConversionSoftware,'dcm2nii') || strcmp(parms.ConversionSoftware,'dcm2niix')) && isfield(parms,'ConversionSoftwareVersion')
+		dcm2niixVersion = parms.ConversionSoftwareVersion;
```

Comment:
No guessing. There is the software name and version. So that we're not limited to dcm2nii.

Reply:
But why not using the pre-existing field?

Also, there's also a comment above this one

Reply:
Because there are no pre-existing field for conversionsoftware name and conversion software version. I had to create new fields.

Reply:
I thought we had `impar.dcm2nii_version`. Are you sure we got rid of that? It would be nice to use the same names throughout the import.

Reply:
Yes. we have that. but that's a specific field for dcm2nii. I wanted to have general fields that would describe: (i) what software it was (if other than dcm2nii) and (ii) what was the version of that software (not necessarily dcm2nii). 
So the current single field dcm2nii wasn't really good for that.

So I revamped stuff going from very specific, non-modular, hard to read stuff. To a nice new revamped stuff.

Reply:
OK I just don't understand what other versions we have, we already have a field for ExploreASL software, and we don't use anything else than dcm2niiX? And just so I understand: we still have the same field but you renamed it? Let's move this discussion to the new issue, then you can resolve this conversation :)

Reply:
Yes - but that's talking about the whole import.
And dcm2nii_version is a parameter on input, and it is still there and can be used.

We need a field that will record the software actually used for DICOM2NII.

Reply:
OK then we should create a general single Provenance field, rather than having to search for the specific Provenance names you came up with ;)

You always use the input parameter field as version right?
Otherwise, throughout the processing, we would need to record from each parameter what the user wanted and what the pipeline did in the end... I would only record what the pipeline did.